crates/zed/src/languages/swift/tags.scm

```diff
@@ -0,0 +1,51 @@
+(class_declaration
```

Comment:
We do not use a tags query, however are analogous to the `outline.scm` queries we have.